pkg/tracing/http_test.go

```diff
@@ -87,7 +94,16 @@ func TestHTTPSpanMiddleware(t *testing.T) {
 	}
 
 	spans := reporter.Flush()
-	if len(spans) != 1 {
-		t.Errorf("Got %d spans, expected 1", len(spans))
+	if len(spans) != 2 {
+		t.Errorf("Got %d spans, expected 2", len(spans))
+	}
+	if spans[0].TraceID.String() != "821e0d50d931235a5ba3fa42eddddd8f" {
+		t.Error("Span 1's TraceID is not correct")
```

Comment:
Want to follow the usual scheme of

```golang
if got, want := spans[0].TraceID.String(); "821e0d50d931235a5ba3fa42eddddd8f" {
  t.Errorf("spans[0].TraceID = %v, want %v, got, want)
}
```

That generally helps parsing test results quickly as you know what to expect.